google-http-client/src/main/java/com/google/api/client/http/HttpResponse.java

```diff
@@ -331,7 +331,8 @@ public InputStream getContent() throws IOException {
           if (!returnRawInputStream
               && contentEncoding != null
               && contentEncoding.contains("gzip")) {
```

Comment:
not a new bug, but "contains" is incorrect. This should be equals or perhaps equalsIgnoreCase with a trim, or something like that. However it shouldn't fire on foozipbar.

filed https://github.com/googleapis/google-http-java-client/issues/842